pkg/sfu/buffer/helpers.go

```diff
@@ -35,22 +33,23 @@ var (
 */
 type VP8 struct {
 	FirstByte byte
+	S         bool
 
-	PictureIDPresent int
-	PictureID        uint16 /* 8 or 16 bits, picture ID */
-	MBit             bool
+	I         bool
+	M         bool
+	PictureID uint16 /* 8 or 16 bits, picture ID */
```

Comment:
Cleaning these up and making them look like fields in RFC rather than super descriptive names.